include/internal/catch_capture.hpp

```diff
@@ -130,6 +130,12 @@
 #define INTERNAL_CATCH_INFO( macroName, log ) \
     Catch::ScopedMessage INTERNAL_CATCH_UNIQUE_NAME( scopedMessage )( Catch::MessageBuilder( macroName##_catch_sr, CATCH_INTERNAL_LINEINFO, Catch::ResultWas::Info ) << log );
 
+///////////////////////////////////////////////////////////////////////////////
+#define INTERNAL_CATCH_UNSCOPED_INFO( macroName, log ) \
+    do { \
```

Comment:
Is there a particular reason why it is in `do {...} while(false)` statement? Can be just the
```c++
Catch::getResultCapture().emplaceUnscopedMessage( Catch::MessageBuilder( macroName##_catch_sr, CATCH_INTERNAL_LINEINFO, Catch::ResultWas::Info ) << log )
```
without the semicolon to prevent latest clang warning `-Wextra-semi-stmt`

Reply:
Yes, it can, but I tend to wrap all expression macros with `do..while(0)` to be on safe side about semicolon for later modifications. It eventually comes down to personal/project coding conventions though, pretty much like preference of using or not using brackets for one-liner `if`s, `while`s,  `for`s etc.

Shall I drop do-while?

Reply:
Done.